src/App.js

```diff
@@ -11,6 +11,8 @@ import Error from "./components/Error";
 import { AuthProvider } from "./context/AuthContext";
 import Register from "./pages/Register";
 import "bootstrap/dist/css/bootstrap.min.css";
+import CreateShop from "./pages/CreateShop";
+import Browse from "./pages/Browse";
 import Login from "./pages/Login";
 import { ServerHostnameProvider } from "./context/ServerHostnameContext";
 
```

Comment:
I think the `Create` element needs to be changed to `CreateShop` in the App page right?